tools/topology/topology2/cavs-sdw.conf

```diff
@@ -13,6 +13,7 @@
 <virtual.conf>
```

Comment:
Should add comment about MCPS impact. It's +4 MCPS in MTL platfrom for IIR, FIR, DRC in passthrough. More when programmed to process.